tailwind.config.js

```diff
@@ -58,6 +61,18 @@ module.exports = {
 
       'bottom-inset': 'inset 0px -4px 4px rgba(0, 0, 0, 0.06)',
       'top-inset': 'inset 0px 4px 4px rgba(0, 0, 0, 0.06)',
+
+      'yellow-primary-2px': `0px 0px 0px 2px ${yellowPrimary}`,
```

Comment:
These new shadows have been very helpful for the cards states. Before we did a load of border re-jigging but now hovers for cards in specific states just use shadows 👍

Reply:
💚 opinion time. I'm not a massive fan of using a shadow when it looks like and behaves like a border, we may want to add a shadow to this element later only to find out that we will be overriding the "border" by doing so. 

How bad was the juggling? David did a workaround using a pseudo element before 👀

Reply:
The juggling was bad. We had fake transparent borders in places which then wouldn't work well with image content which needed to be flush and the implementation was much worse than this.

I don't feel the same way as you do though, shadows are just another tool which can be utilised to create a look. I recommend that once this is merged in you give it a go and see if you can get borders to function as it does here and then I can see how we could use borders instead 👍

Reply:
I also tried using a pseudo-element and it looked horrible with clipping etc so if you wanna try that again you can, maybe I was missing something 👍  

(although I'd argue that your original comment about pretending to be a border is no different for a pseudo-element vs a shadow).

Reply:
It's just a 💚  so no biggie. Its quite opinionated and if its saving a load of hassle then its no problem.

I would disagree with your argument though! The pseudo element is still using a border and not taking the place of any shadow.